### PR TITLE
Display embedded python version in the status page

### PR DESCRIPTION
**Pull request by hush-hush**
### What does this PR do?

Display embedded python version in the status page

### CODE REVIEW

**Review by derekwbrown — APPROVED** [review]
lgtm

File: pkg/metadata/host/host.go [hunk 1]
```diff
@@ -68,11 +68,9 @@ func GetMeta() *Meta {
 	return getMeta()
 }
 
-// getPythonVersion returns the version string as provided by the embedded Python
-// interpreter. The string is stored in the Agent cache when the interpreter is
-// initialized (see pkg/collector/py/utils.go), an empty value is expected when
-// using this package without embedding Python.
```
Comment (olivielpeau) [inline pkg/metadata/host/host.go]: why did you remove part of the comments here? They look useful to me.
